### PR TITLE
feat: add globmatch

### PR DESCRIPTION
**Pull request by feloraaa**
Fix: https://github.com/casbin/SwiftCasbin/issues/23

Signed-off-by: feloraaa <704596810@qq.com>

### CODE REVIEW

**Comment by hsluoyz** [comment]
@xiaobeiswift plz review

**Comment by hsluoyz** [comment]
@meekotherogue plz review

**Review by meekotherogue — CHANGES_REQUESTED** [review]

File: Tests/CasbinTests/UtilsTests/KeyMatchTests.swift [hunk 1]
```diff
@@ -33,6 +33,16 @@ final class KeyMatchTests: XCTestCase {
...
+    func testglobMatch() {
+        XCTAssertTrue(Util.globMatch("/foo", "/foo"))
+        XCTAssertTrue(Util.globMatch("/foo", "/foo*"))
+        XCTAssertFalse(Util.globMatch("/foo", "/foo/*"))
+        XCTAssertFalse(Util.globMatch("/foo/bar", "/foo"))
+        XCTAssertFalse(Util.globMatch("/foo/bar", "/foo*"))
+        XCTAssertTrue(Util.globMatch("/foo/bar", "/foo/*"))
+        XCTAssertFalse(Util.globMatch("/foobar", "*/foo"))
+        XCTAssertFalse(Util.globMatch("/prefix/foobar", "*/foo/*"))
+    }
```
[the remark this answers is not in the document]
Comment (hsluoyz) [reply Tests/CasbinTests/UtilsTests/KeyMatchTests.swift:45]: @feloraaa

**Comment by hsluoyz** [comment]
Replaced by: https://github.com/casbin/SwiftCasbin/pull/26

**Pull request closed without merging**